pom.xml

```diff
@@ -145,6 +146,11 @@
                 <artifactId>jetty-servlet</artifactId>
                 <version>${version.jetty}</version>
             </dependency>
+            <dependency>
+                <groupId>mysql</groupId>
+                <artifactId>mysql-connector-java</artifactId>
+                <version>${version.mysql}</version>
```

Comment:
Should this not still default to `<scope>test</scope>` unless overridden?  Only the hibernate connectors require this dependency at runtime.

Reply:
No component is using it for testing as of today. Dependency scope:
* `elide-blog-example` - runtime
* `elide-blog-example-resteasy` - runtime
* `elide-hibernate3-mysql-example` - runtime
* `elide-standalone` - compile

And in `elide-standalone` tests are actually using `InMemoryDataStore`

Reply:
Note - `jetty-server` and `jetty-servlet` have similar usage and are not in test scope